x/cellarfees/client/rest/rest.go

```diff
@@ -0,0 +1,21 @@
+package rest
+
+import (
+	"github.com/gorilla/mux"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/rest"
+)
+
+// REST query and parameter values
+const (
+	MethodGet = "GET"
+)
+
+// RegisterRoutes registers the cellarfees module REST routes.
+func RegisterRoutes(clientCtx client.Context, rtr *mux.Router, storeName string) {
+	r := rest.WithHTTPDeprecationHeaders(rtr)
+	r.HandleFunc(
+		"/cellarfees/apy", QueryAPYRequestHandlerFn(storeName, clientCtx),
```

Comment:
Should we construct the argument to `HandleFunc` using the `types` values as we do elsewhere (`types.QuerierRoute` and `types.QueryAPY`)?